jobs/v4/src/main/java/com/google/samples/JobSearchCreateCompany.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.samples;
+
+// [START job_search_create_company]
```

Comment:
Can you add `_beta` to the end of all the region tags, since this is currently beta. Will also help with tracking when it goes GA. 

`// [START job_search_create_company_beta]`

It'd be nice to also put all the files under:
`jobs/beta`
Then when they go GA we can move them to `jobs/cloud-client`

Reply:
Will do